armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -313,6 +314,11 @@ def fromUserSettings(self, cs):
         This is not required; these options can alternatively be set programmatically.
         """
         self.kernelName = cs["neutronicsKernel"]
+        self.runDir = os.path.join(
+            armi.FAST_PATH, f"{cs.caseTitle}-{self.label}-{armi.context.MPI_RANK}"
+        )
+        if cs["restartNeutronics"]:
+            self.isRestart = True
```

Comment:
What doe these lines doe that the line below does not?

Reply:
lol not much!